dnsvizwww/templates/base_site.html

```diff
@@ -42,6 +42,11 @@ <h2 class="site-subtitle"><a href="/"><img src="{% static "images/logo-dnsviz.pn
 			<ul>
 				<li><a href="https://www.sandia.gov/"><img src="{% static "images/logo-sandia.png" %}" alt="Sandia National Laboratories" /></a></li>
 				<li><a href="https://www.verisign.com/"><img src="{% static "images/logo-verisign.png" %}" alt="Verisign" /></a></li>
+      </ul>
+      </div>
+      <div id="footer-buttons">
```

Comment:
I'm not an HTML/CSS expert, but I think element IDs are supposed to be unique throughout the doc.

Reply:
Yeah, you're right about that.  I was a bit on autopilot when I did that and was reading that as `class`, not `id`.

You know the CSS setup for the site better than I do.. if you want to suggest the correct way to set up those elements to get the right effect I'm happy to take your advice.

Reply:
As a non-Web designer, I'm actually not sure the best way to fix this... 😬 I know that I'm the one that brought it up.  Maybe change the CSS, so footer buttons is a class instead of an id?

Reply:
I'll have a closer look at the CSS later and see what makes sense.